### PR TITLE
Adding Bis-SNP v0.82.2. The bioconda code is built upon the picard code

### PR DESCRIPTION
**Pull request by karl616**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
Hi @karl616 thanks for your contribution. Two small comments.

File: recipes/bis-snp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  url: https://downloads.sourceforge.net/project/bissnp/BisSNP-{{ version }}/BisSNP-{{ version }}.jar
+  sha256: f15358c89bfdaff04c407ec382940c70cfa00857e88f7e917aa4ba11345f4e52
+
+build:
+  number: 0
+  skip: False
+
+requirements:
+  run:
+    - java-jdk ==7.0.91
```
[the remark this answers is not in the document]
Comment (karl616) [reply recipes/bis-snp/meta.yaml]: There is only java 8 available for openjdk. Bis-SNP requires an java 7 or older. I tried to add java 7 to openjdk:

https://github.com/conda-forge/openjdk-feedstock/pull/26

but it doesn't pass the automatic tests. So until this is solved I need to use java-jdk

File: recipes/bis-snp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  version: {{ version }}
+
+source:
+  fn: BisSNP-{{ version }}.jar
+  url: https://downloads.sourceforge.net/project/bissnp/BisSNP-{{ version }}/BisSNP-{{ version }}.jar
+  sha256: f15358c89bfdaff04c407ec382940c70cfa00857e88f7e917aa4ba11345f4e52
+
+build:
+  number: 0
+  skip: False
```
[the remark this answers is not in the document]
Comment (karl616) [reply recipes/bis-snp/meta.yaml]: The skip line? or the whole section?

**Comment by karl616** [comment]
Hi @bgruening Thanks for the feedback. I cleaned out the skip line and as soon as I have an older version for java in openjdk I will change this as well

**Pull request merged**